unit_test/entities/test_esdl_object.py

```diff
@@ -37,14 +39,26 @@ def test_EsdlAssetObject(self):
         consumers = self.esdl_object.get_all_assets_of_type(consumer)
         pipes = self.esdl_object.get_all_assets_of_type(pipe)
         # Act
-        asset_producer = producers[0].convert_esdl()
```

Comment:
I think that this should be a separate commit? You are removing the convert_esdl method from the EsdlAssetObject? 

@MichielTukker

Reply:
the convert method has been moved, so this test does not work anymore and the correct calls have been added